_data/i18n/en.yml

```diff
@@ -63,9 +63,9 @@ homepage_featurette_network_performance_title: Network Performance
 homepage_featurette_network_performance_description: |
   Yarn efficiently queues up requests and avoids request waterfalls in order
   to maximize network utilization.
-homepage_featurette_npm_and_bower_title: Multiple Registries
-homepage_featurette_npm_and_bower_description: |
-  Install any package from either npm or Bower and keep your package workflow
+homepage_featurette_npm_title: Multiple Registries
```

Comment:
Should we just drop the whole "Multiple Registries" section? Without mentioning Bower, the section heading "Multiple Registries" doesn't make much sense:

> ### Multiple Registries
> Install any package from npm and keep your package workflow the same.